geth/mixins.py

```diff
@@ -154,8 +155,12 @@ def __init__(self, *args, **kwargs):
 
         super(LoggingMixin, self).__init__(*args, **kwargs)
 
-        stdout_logger = get_file_logger('geth-stdout', stdout_logfile_path)
-        stderr_logger = get_file_logger('geth-stderr', stderr_logfile_path)
+        stdout_logger = get_file_logger(
+            'geth-stdout', stdout_logfile_path, log_format='%(message)s'
+        )
```

Comment:
Since this is the only place that `get_file_logger` is called, I'm not sure that it makes sense to have a `default_log_format` in the `get_file_logger` method instead of just changing the message format in `get_file_logger`. Unless I'm reading this wrong, the `log_format` in `get_file_logger` _looks_ like it is configurable, but it's not really from a user standpoint. I think a better option here might be to add a check to see if `log_format` gets passed in with the kwargs for `__init__`, and then pass it along to `get_file_logger`. What do you think? I'm always open to pushback :)

Reply:
Yes, that all makes sense to me! Thank you for looking at this. I will make that happen.

Reply:
No one  would be calling `get_file_logger()` outside of this package, right? If the method was named `_get_file_logger()`, it would feel a little safer

Reply:
I don't think so. I like the `_get_file_logger()` change. Good idea!

Reply:
Oh, wait I guess I will push back a little here on something:

There is not much reason for the user to set to want to any format when using `LoggingMixin`. If you look at the issue link, the main problem is that whatever format you give it does not really matter, because you are logging already logged log lines.  So really, what needs to happen to for the mixin to just not use format. (only pass in `(message)`).. If nothing else is calling that `get_file_logger()` method, then, it can be the default there. I didn't change it there because `LoggingMixin` is the one who needs the format to be gone.

Reply:
That works. I'm good changing the default in `get_file_logger()` to be `%(message)s`. Do you think there is a reason to still make it configurable via `LoggingMixin` initialization? I think it would be nice just in case someone is doing some log parsing with the oddly formatted messages, but I really don't think it's a very large use case so I could see either way.

Reply:
It might confuse users. I think I would assume that `log_format` actually adjusts geth's log format rather than the logger that logs geth's logs

Reply:
Got it. Yeah, that makes sense. I'm not super familiar with Python's logging, but it doesn't look like there's an easy way to parse out the message, right?